cartography/data/jobs/cleanup/gcp_sql_cleanup.json

```diff
@@ -6,6 +6,18 @@
 			"iterationsize": 100,
 			"__comment__": "Delete GCP Cloud SQL Users that no longer exist and detach them from all previously connected nodes."
 		},
+		{
+			"query": "MATCH (:GCPSQLDatabase)<-[r:HAS_DATABASE]-(:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE r.lastupdated <> $UPDATE_TAG WITH r LIMIT $LIMIT_SIZE DELETE (r) return COUNT(*) as TotalCompleted",
+			"iterative": true,
+			"iterationsize": 100,
+			"__comment__": "Remove GCP GCPSQLDatabase-to-Instance relationships that are out of date."
+		},
+		{
+			"query": "MATCH (d:GCPSQLDatabase)<-[:HAS_DATABASE]-(:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE u.lastupdated <> $UPDATE_TAG WITH u LIMIT $LIMIT_SIZE DETACH DELETE (u) return COUNT(*) as TotalCompleted",
```

Comment:
variable `u` does not exist in the query. it should be `d`

Reply:
change the variable name